src/test/transaction_tests.cpp

```diff
@@ -158,7 +158,7 @@ std::vector<unsigned int> ExcludeIndividualFlags(unsigned int flags)
     std::vector<unsigned int> flags_combos;
     for (unsigned int i = 0; i < mapFlagNames.size(); ++i) {
         const unsigned int flags_excluding_i = TrimFlags(flags & ~(1U << i));
-        if (flags != flags_excluding_i && std::find(flags_combos.begin(), flags_combos.end(), flags_excluding_i) != flags_combos.end()) {
+        if (flags != flags_excluding_i && std::find(flags_combos.begin(), flags_combos.end(), flags_excluding_i) == flags_combos.end()) {
```

Comment:
Any reason to re-invent the wheel and avoid `std::set`?

Reply:
Nope, I could change it to set.

Reply:
I've changed it to use a `std::set`